examples/zone-printer/README.md

```diff
@@ -0,0 +1,247 @@
+PROJECT=<your-project-name>
+```
+
+### 0.2 `gcloud` installed and initialized
+
+You can install `gcloud` by following the instructions at
+[Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
+initialize it by running the following commands:
+
+```shell
+gcloud init # Set $PROJECT as the default project
+gcloud auth login
+gcloud auth application-default login
+```
+
+### 0.3 kubectl
+
+You can install `kubectl` by running:
+
+```shell
+gcloud components install kubectl
+```
+
+### 0.4 Kubernetes clusters on GCP
+
+#### Existing clusters
+
+> **Note:**: If you don't already have Kubernetes cluster on GCP,
+skip to the next subsection on [creating new clusters](#creating-new-clusters).
+
+`kubemci` requires Kubernetes clusters that are v1.8.1 or newer. Please
+ensure that all the clusters you are using in this tutorial satisfy the
+minimum version requirement. You can check your cluster version using
+the `kubectl version` command.
+
+If you already have clusters with their credentials in your local
+kubeconfig (`$HOME/.kube/config` by default) file, you need to ensure
+that you either only have clusters you are going to use for this
+tutorial in your default kubeconfig file or create a new kubeconfig
+file that just contains credentials for those clusters. Assuming that
+the clusters you are going to use for this tutorial are called
+`us-east`, `eu-west` and `asia-east`.
+
+List the contexts available (note the names of the ones you wish to
+use with kubemci)
+
+```shell
+$ kubectl config get-contexts -o name
+us-east
+eu-west
+asia-east
+i-dont-care
+doesnt-exist
+```
+
+Now iterate over the contexts you identified, making it the context
+in-use and then extracting that context to its own file.
+
+```shell
+# First, do this for the us-east one.
+kubectl config use-context us-east
+kubectl config view  --minify --flatten > mciuseast
+
+# Second, do this for the eu-west one.
+kubectl config use-context eu-west
+kubectl config view --minify --flatten > mcieuwest
+
+# Finally, do this for the asia-east one.
+kubectl config use-context asia-east
+kubectl config view --minify --flatten > mciasiaeast
+```
+
+Combine (flatten) all of these extracted contexts to create a
+single kubeconfig file with just the contexts we care about
+
+```shell
+# Use KUBECONFIG to specify the context filenames we created in
+# the previous step
+KUBECONFIG=mciuseast:mcieuwest:mciasiaeast kubectl config view \
+  --flatten > mcikubeconfig
+```
+
+#### Creating new clusters
+
+You need at least two Kubernetes clusters in two different GCP zones
+to verify that `kubemci` works. Clusters must be v1.8.1 or newer.
+Let's create three GKE clusters and get their credentials for the
+purposes of this tutorial:
+
+```shell
+# Create a cluster in us-east and get its credentials
+gcloud container clusters create \
+    --cluster-version=1.8.1-gke.0 \
+    --zone=us-east4-a \
+    cluster-us-east
+KUBECONFIG=./mcikubeconfig gcloud container clusters get-credentials \
```

Comment:
Adding a line break here will help in readability. You can add 2 line breaks after this to separate different clusters

Reply:
Makes sense. Done.